<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>live-share</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string>Uploads</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="categoryLineEdit">
          <property name="placeholderText">
           <string>Enter current upload category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>707</width>
             <height>612</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0">
            <item row="0" column="1">
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>henry/ksp/2017-12-29_0001.png</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDialogButtonBox" name="buttonBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel</set>
              </property>
              <property name="centerButtons">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="urlLabel">
          <property name="text">
           <string>Service URL</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="urlLineEdit">
          <property name="placeholderText">
           <string>Enter base URL of live-share service</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="imageLabel">
          <property name="text">
           <string>Image Hotkey</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QKeySequenceEdit" name="imageKeySequenceEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="videoLabel">
          <property name="text">
           <string>Video Hotkey</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QKeySequenceEdit" name="videoKeySequenceEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="ffmpegLabel">
          <property name="text">
           <string>FFmpeg Executable</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="ffmpegLineEdit">
            <property name="placeholderText">
             <string>ffmpeg</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ffmpegFileDialogButton">
            <property name="text">
             <string>Select</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonFollowStyle</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Max. Image Edge Length</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="maxImageEdgeLengthSpinBox">
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="suffix">
           <string>px</string>
          </property>
          <property name="minimum">
           <number>400</number>
          </property>
          <property name="maximum">
           <number>4000</number>
          </property>
          <property name="value">
           <number>800</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Max. Video Edge Length</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="maxVideoEdgeLengthSpinBox">
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="suffix">
           <string>px</string>
          </property>
          <property name="minimum">
           <number>40</number>
          </property>
          <property name="maximum">
           <number>4000</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Video Frame Rate</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDoubleSpinBox" name="frameRateDoubleSpinBox">
          <property name="suffix">
           <string>Hz</string>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
